docs/topics/feed-exports.rst

```diff
@@ -307,6 +315,15 @@ Default: ``{}``
 A dict containing additional feed storage backends supported by your project.
 The keys are URI schemes and the values are paths to storage classes.
 
+.. setting:: FEED_STORAGE_FTP_ACTIVE
+
+FEED_STORAGE_FTP_ACTIVE
+-----------------------
+
+Default: ``False``
+
+Whether to use active mode when exporting feeds to a FTP server.
```

Comment:
Maybe "active mode" here should also link to https://stackoverflow.com/a/1699163 ?